### ISSUE TITLE
EVEX prefix (62h), used by AVX-512 and other new instruction sets, is listed as invalid in 64-bit mode

### ISSUE THREAD

**Issue by Googulator** [mazegen/x86reference#22]
62h is no longer invalid on CPUs that support AVX-512, AVX10 or APX. Instead, it's the first byte of the EVEX prefix.

**Comment by mazegen** [comment]
Yes, VEX encoding is not supported at all now, I'm working on it.

* 62h is the first byte of EVEX prefix
* C4h and C5h is the first byte of VEX prefix

**Comment by mazegen** [comment]
See #8.
